### PR TITLE
Create contribution guidelines and code of conduct

### PR DESCRIPTION
**Pull request by jhsi517**
The following files are created to resolve issue #8

Create CONTRIBUTING.md

Create reactjs-guidelines.md

Create C#-guidelines.md

Create CODE_OF_CONDUCT.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `master` or development branch in the repository.
+
+3. **Isolate the problem** &mdash; create a [reduced test
+   case](http://css-tricks.com/reduced-test-cases/) and a live example.
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
```
Comment (JedLJRobertson) [inline CONTRIBUTING.md]: Can this section just become "Please fill out the bug report issue template" or something?

Comment (jhsi517) [reply CONTRIBUTING.md]: True, I will update this section.

Comment (jhsi517) [reply CONTRIBUTING.md]: Updated. Please review. :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+   ```bash
+   git pull [--rebase] upstream <dev-branch>
+   ```
+
+6. Push your topic branch up to your fork:
+
+   ```bash
+   git push origin <feature/[id]-some-feature>
+   OR
+   git push origin <fix/[id]-some-bug>
```
Comment (HongShi10) [inline CONTRIBUTING.md]: I think we should go with the convention bug/#id-some-bug

Comment (jhsi517) [reply CONTRIBUTING.md]: Have updated the naming conventions :) Please review.

**Pull request merged**